Bindings/Java/tests/TestBasics.java

```diff
@@ -23,6 +23,12 @@ public static void main(String[] args) {
 
     m.addComponent(pj);
 
+    // Change from ArrayPtrs<Object*> being non-const const.
```

Comment:
As long as it is meant to be here I am happy to merge it. I imagine you will you or Ayman will be testing all the sets after dev week. So you no point adding better documentation now. I will merge it.